introductions/WinsonChen.txt

```diff
@@ -0,0 +1,11 @@
+Name: Winson Chen
+Pronouns: he/him
+Year: Junior
+Major: Computer Engineering
+
+I learned C and C++ from my classes at the University of Michigan.
+I also learned the PERN stack during my internship at CodePath during the summer of 2022.
+
+Outside of school, I enjoy watching anime and playing Genshin Impact.
```

Comment:
Wow, seems like we have similar interests!

Reply:
What a coincidence!